client/state/privacy-policy/actions.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ *
+ * @format
+ */
+import { PRIVACY_POLICY_ADD, PRIVACY_POLICY_REQUEST } from 'state/action-types';
+
+/**
+ * Internal dependencies
+ */
+
+export const requestPrivacyPolicy = () => ( {
+	type: PRIVACY_POLICY_REQUEST,
+} );
+
+export const privacyPolicyReceive = entities => ( {
```

Comment:
Can be renamed to `addPrivacyPolicy`

Reply:
sure!

Reply:
Actually, there is a `addPrivacyPolicy()` also in the data-layer implementation so I'm considering keeping this name.

Reply:
You can use aliases :). eg `import { addPrivacyPolicy as addPolicy } from 'state/privacy-policy/actions';`